### ISSUE TITLE
Environment variable is not being passed to the extension host

### ISSUE THREAD

**Issue by ramya-rao-a** [Microsoft/vscode#65113]
Related to https://github.com/Microsoft/vscode-go/issues/2026 by @lrstanley

@lrstanley is on a Windows machine and has an environment variable called `GOPROXY` which is available in both Git bash and the Windows command prompt.

When running VS Code (either directly or from the Git bash or the Windows command prompt), the variable is being passed to VS Code just fine as it is available when echoing in the integrated terminal. 

But the same is not being passed to the Go extension. We added console.log statements in the extension code to print out `process.env.GOPROXY` and it wasnt available.

Therefore, logging this issue here.

**Comment by isidorn** [comment]
I think this is a duplicate, but @weinand would know better

**Comment by weinand** [comment]
This doesn't seem to be debug related.

**Comment by alexandrudima** [comment]
@ramya-rao-a I cannot reproduce. Here is what I have tried:

* I have defined an environmental variable `TEST_65113`:

![image](https://user-images.githubusercontent.com/5047891/50091900-e963b680-020c-11e9-8996-7bb946bc61c3.png)

* `package.json`:
```
{
    "name": "65113",
    "publisher": "alex",
    "version": "1.0.0",
    "engines": {
        "vscode": "^1.31.0"
    },
    "activationEvents": ["*"],
    "main": "index.js"
}
```

* `index.js`
```
console.log(JSON.stringify(process.env['TEST_65113']));
```

I can then see that the value is available:
![image](https://user-images.githubusercontent.com/5047891/50092026-4495a900-020d-11e9-9f64-b934cec90c3c.png)

**Comment by lrstanley** [comment]
Environment variable passthrough works. For some reason using GOPROXY
itself doesn't. My first thought was because vscode has global proxy
configurations, maybe it strips out *PROXY environment variables from the
object passed to extensions.

When I am back in the office I will test naming it something else entirely,
or naming it something like TESTPROXY.

**Comment by alexandrudima** [comment]
@lrstanley I also cannot reproduce when naming the environment variable as `GOPROXY`. The extension I've written receives the environmental variable correctly.

**Comment by lrstanley** [comment]
I can still seem to replicate, specifically with `GOPROXY` in relation to vscode-go. If you have a sample vsix file I could use to test, I'd imagine that would probably work. I've attempted `TESTPROXY`, and this is when I noticed the testing that I was told to do in https://github.com/Microsoft/vscode-go/issues/2026 may have not been correct. After testing, I cannot print any of the environment variables from `process.env` within the vscode-go extension section that I was told to modify when debugging.

I will follow up in https://github.com/Microsoft/vscode-go/issues/2026 to troubleshoot on the vscode-go side a bit more.
